examples/basic-examples/test/e2e/integration/smoke/error-page/step_definition.js

```diff
@@ -0,0 +1,15 @@
+/* global cy */
+
+import { Then } from 'cypress-cucumber-preprocessor/steps'
+
+Then(`the example 404 error content is added`, () => {
+  cy.get('.app-error-intro').should('contain', 'This is a custom 404 error message')
+  cy.get('.app-error-main').should('contain', 'Your custom main 404 error message')
+  cy.get('.app-error-cta .rpl-link__inner').should('contain', 'Go to 500 error')
+})
+
+Then(`the example 500 error content is added`, () => {
```

Comment:
instead of making it hardcoded in the step definition, pass the message in the feature file eg:

`Then the app error page title should be "This is a custom 500 error message"`
`Then the app error page body should be "This is a custom 500 error message"`
`Then the app error page cta button should contain "Go to 404 error"`

That way the step definition can be reused for both tests

Reply:
To test the HTTP error code you can do a seperate request using 

`Given a request is made to {string}`
`Then the response code should be {int}`